src/assets/scripts/modules/download-filter/download-filter.js

```diff
@@ -0,0 +1,59 @@
+import List from 'list.js';
+
+const component = document.querySelector('[data-downloads]');
+const filter = document.querySelector('[data-downloads-filter]');
+const sort = document.querySelector('[data-downloads-sort]');
+const formatButtons = Array.from(document.querySelectorAll('[data-downloads-format]'));
```

Comment:
I like to actually be much more explicit about when I reference DOM elements. So that I can easily make the distinction between variables that have a JS value, and variables that refer to DOM elements.
In fact, I put a `$` sign in front of the variable name to make the distinction. So I would have:
```
const $rootComponent
const $filterForm
const $sortButton
const $formatButtons
```

Reply:
refactored to use that syntax